pages/account/customer/iam-policies-api/guide.en-gb.md

```diff
@@ -0,0 +1,540 @@
+- **urn**: Resource group URN to use on the policy
+- **readOnly**: Cannot be modified if it's true. The default resource groups are read-only
+- **name**: The resource group name
+- **owner**: The resource group owner (NIC Handle)
+- **resources**: 
+    - If details = **false**: Array of resources UUID
+    - If details = **true**: The resources will be expanded with their attributes (as the result we have via the resource API)
+- **createdAt**: Creation date of the resource group
+- **updateAt**: Last update date of the resource group
+
+#### Create a resource group
+
+Create a resource group with this API:
+
+|**Method**|**Path**|**Description**|
+| :-: | :-: | :-: |
+|POST|/iam/resourceGroup|Create a new resource group|
+The following example creates a resource group called "*Test\_Envrionment*" containing two resources:
+
+Error rendering macro 'code': Invalid value specified for parameter 'com.atlassian.confluence.ext.code.render.InvalidValueException' 
+
+```json
+{
+  "name": "Test_environment",
+  "resources": [
+    {
+      "id": "b10611a9-7cc1-40b7-8475-18aabcdffc62"
+    },
+    {
+      "id": "53f815b1-c020-4e5d-8239-fbbcaaca1742"
+    }
+  ]
+}
+
+```
+
+### Action
+
+Policies contain a list of **Actions** that will be allowed or denied to users.
+
+These actions are specific to every product, such as rebooting a database, ordering an upgrade, restoring a snapshot, etc.
+
+#### API definition
+
+<https://api.ovh.com/console-preview/?section=%2Fiam&branch=v2#get-/iam/reference/action>
+
+|**Method**|**Path**|**Description**|
+| :-: | :-: | :-: |
+|GET|/iam/reference/action|Retrieve all actions|
+
+#### **Example**
+
+List all the actions available for policies with the API.
+
+Here's an example:
+
+**action example**
+
+```json
+{
+    "action": "vps:apiovh:reboot",
+    "description": "Request a reboot of the machine",
+    "resourceType": "vps",
+    "categories": [
+        "OPERATE"
+    ]
+}
+```
+
+This action is "*vps:apiovh:reboot*", it targets the ability to reboot a VPS.
+
+The call to /iam/reference/action API will list **all of the available** actions (WARNING: this contains a huge amount of items).
+
+It's strongly recommended to specify the **resourceType** as a query-string parameter for this API (see the next section).
+
+#### Action attributes
+
+- **action**: The action itself
+- **description**: The action description
+- **resourceType**: The resource type targeted by this action
+- **categories**: The categories of this action (CREATE, READ, EDIT, OPERATE, DELETE)
+
+### Resource types
+
+#### **API definition**
+
+|**Method**|**Path**|**Description**|
+| :-: | :-: | :-: |
+|GET|/iam/reference/resource/type|Retrieve all resource types|
+
+#### **Example**
+
+Here's a part of the output:
+
+**/iam/reference/resource/type**
+
+```json
+[
+    "account",
+    "packXdsl",
+    "cloudDB",
+    "loadbalancer",
+    "mxplanAccount",
+    "publicCloud",
+    "voipSmsvn",
+    "vps",
```

Comment:
```suggestion
    "vps"
```

Reply:
This one was on purpose to specify that there is actually way more output here. At first we put "..." but as it's a json error it was highlited in red